src/dnx.mono.managed/EntryPoint.cs

```diff
@@ -58,6 +58,19 @@ public static int Main(string[] arguments)
         // Set the default lib path to be next to the entry point location
         Environment.SetEnvironmentVariable(EnvironmentNames.DefaultLib, Path.GetDirectoryName(typeof(EntryPoint).Assembly.Location));
         Environment.SetEnvironmentVariable(EnvironmentNames.ConsoleHost, "1");
+        
+        var p = (int)Environment.OSVersion.Platform;
+        string os;
+        if ((p != 4) && (p != 6) && (p != 128))
```

Comment:
I was using the old way we did it, https://github.com/aspnet/dnx/commit/79b8ee24e8ea5a241687756a28c0be1a057ecb1b#diff-c889083e2a55b6b0e4ad6cc6e256f8c4L404